src/mame/drivers/sandscrp.cpp

```diff
@@ -142,19 +141,26 @@ class sandscrp_state : public driver_device
 
 uint32_t sandscrp_state::screen_update(screen_device &screen, bitmap_ind16 &bitmap, const rectangle &cliprect)
 {
+	int i;
```

Comment:
The loop counters are completely independent and should be scoped to the loops they control.  It isn't a single logical value that persists from one loop to the next.